components/NavigationBar/Navigation.styled.js

```diff
@@ -58,7 +58,7 @@ export const NavLink = styled.a`
   }
 
   ${(props) =>
-    props.$active &&
+    props.active &&
```

Comment:
please change this to $active, since this way the props are not forwarded to the actual html elements.